docs/production/settings.md

```diff
@@ -79,17 +83,19 @@ configuration along with your other Zulip server configuration.
 ### Miscellaneous server settings
 
 Zulip has dozens of settings documented in the comments in
-`/etc/zulip/settings.py`; you can review
-[the latest version of the settings.py template][settings-py-template]
-if you've deleted the comments or want to check if new settings have
-been added in more recent versions of Zulip.
+`/etc/zulip/settings.py`; you can review [the latest version of the
+settings.py template][settings-py-template], and we recommend
+[carefully updating your `/etc/zulip/settings.py`][update-settings] to
+fold in new configuration settings and documentation.
```

Comment:
Maybe "to fold in the inline comment documentation for new settings"?

I want to make clear that it's not a functional step.